ui/exportSettings.glade

```diff
@@ -327,6 +327,24 @@ Set export parameters</property>
                     <property name="top-attach">0</property>
                   </packing>
                 </child>
+
+		<child>
+                  <object class="GtkCheckButton" id="cbExportLayers">
+                    <property name="label" translatable="yes">Export all layers as separate pages</property>
+                    <property name="visible">True</property>
+                    <property name="can_focus">True</property>
+                    <property name="receives_default">False</property>
+                    <property name="tooltip_text" translatable="yes">If enabled, all layers will be rendered successively as different pages, like for a powerpoint or LaTeX beamer document. The resulting PDF file can be used for a presentation.</property>
```

Comment:
I understand the label as "each layer will occupy a different page". This is not what the feature does. I know it's hard to give a concise descritpion of this feature. IMO, an undescriptive sentence is better than a misleading one:
```suggestion
                    <property name="label" translatable="yes">Export as a presentation</property>
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="receives_default">False</property>
                    <property name="tooltip_text" translatable="yes">If enabled, the layers of each page will be added one by one. The resulting PDF file can be used for a presentation.</property>
```